Subtester/Assets/PlayServicesResolver/Editor/play-services-resolver_v1.2.135.0.txt

```diff
@@ -1,8 +0,0 @@
-Assets/PlayServicesResolver/Editor/Google.IOSResolver_v1.2.135.0.dll
-Assets/PlayServicesResolver/Editor/Google.IOSResolver_v1.2.135.0.dll.mdb
-Assets/PlayServicesResolver/Editor/Google.JarResolver_v1.2.135.0.dll
-Assets/PlayServicesResolver/Editor/Google.JarResolver_v1.2.135.0.dll.mdb
-Assets/PlayServicesResolver/Editor/Google.VersionHandler.dll
-Assets/PlayServicesResolver/Editor/Google.VersionHandler.dll.mdb
-Assets/PlayServicesResolver/Editor/Google.VersionHandlerImpl_v1.2.135.0.dll
-Assets/PlayServicesResolver/Editor/Google.VersionHandlerImpl_v1.2.135.0.dll.mdb
```

Comment:
This got removed because you upgraded the plugin, but where are the references to the newer version? 🤔

Reply:
oh wow it means we don't have to download it manually anymore!!!!! Does it mean we don't have to do this either? https://github.com/googlesamples/unity-jar-resolver#plugin-redistribution

Like we don't have to include the unity-jar-resolver package when exporting our package anymore?